infrastructure/transform/main.tf

```diff
@@ -12,11 +12,66 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-resource "azurerm_databricks_workspace" "transform" {
+resource "azurerm_databricks_workspace" "databricks" {
   name                        = "${var.prefix}-${var.environment}-dbks"
   resource_group_name         = var.core_rg_name
   managed_resource_group_name = "${var.prefix}-${var.environment}-rg-dbks"
   location                    = var.core_rg_location
   sku                         = "standard"
   tags                        = var.tags
 }
+
+data "databricks_spark_version" "latest_lts" {
+  spark_version = var.spark_version
+
+  depends_on = [azurerm_databricks_workspace.databricks]
+}
+
+data "databricks_node_type" "smallest" {
+  # Providing no required configuration, Databricks will pick the smallest node possible
+  depends_on = [azurerm_databricks_workspace.databricks]
+}
+
+resource "databricks_cluster" "fixed_single_node" {
+  cluster_name            = "Fixed Job Cluster"
```

Comment:
any potential problems with spaces in names?

Reply:
Didn't seem to cause any problems, and also default clusters that Databricks creates have spaces in their names so should be fine